build-logic/src/main/kotlin/LicenseFileValidation.kt

```diff
@@ -108,9 +107,12 @@ class LicenseFileValidation : DependencyFilter {
     }
     if (!missingApacheMentions.isEmpty() || !missingFullMentions.isEmpty()) {
 
-      throw GradleException(
+      // just a warn, no failure
```

Comment:
Why isn't this a failure? Seems safer to fail long term.

Reply:
Yea I will revert this part as I disabled the gradle plugin for now. Thanks.